api/odigos/v1alpha1/instrumentedapplication_types.go

```diff
@@ -45,12 +45,14 @@ type OtherAgent struct {
 
 // +kubebuilder:object:generate=true
 type RuntimeDetailsByContainer struct {
-	ContainerName  string                     `json:"containerName"`
-	Language       common.ProgrammingLanguage `json:"language"`
-	RuntimeVersion string                     `json:"runtimeVersion,omitempty"`
-	EnvVars        []EnvVar                   `json:"envVars,omitempty"`
-	OtherAgent     *OtherAgent                `json:"otherAgent,omitempty"`
-	LibCType       *common.LibCType           `json:"libCType,omitempty"`
+	ContainerName         string                     `json:"containerName"`
+	Language              common.ProgrammingLanguage `json:"language"`
+	RuntimeVersion        string                     `json:"runtimeVersion,omitempty"`
+	EnvVars               []EnvVar                   `json:"envVars,omitempty"`
+	OtherAgent            *OtherAgent                `json:"otherAgent,omitempty"`
+	LibCType              *common.LibCType           `json:"libCType,omitempty"`
+	CriErrorMessage       *string                    `json:"criErrorMessage,omitempty"`
+	EnvVarsFromDockerFile []EnvVar                   `json:"envVarsFromDockerFile,omitempty"`
```

Comment:
Maybe we should rename this to something like
`EnvFromContainerRuntime` ?
Since containers can be created using other ways from Dockerfile

Reply:
Thanks for sharing Ron, will update